integration-tests/src/test/java/oracle/kubernetes/operator/ItPodsShutdownOptionsEnvDomainCluster.java

```diff
@@ -0,0 +1,292 @@
+   *
+   * @throws Exception if initialization of properties failed.
+   */
+  @BeforeAll
+  public static void staticPrepare() throws Exception {
+    if (FULLTEST) {
+      namespaceList = new StringBuffer();
+      testClassName = new Object() {
+      }.getClass().getEnclosingClass().getSimpleName();
+      initialize(APP_PROPS_FILE, testClassName);
+    }
+  }
+
+  /**
+   * This method gets called before every test. It creates the result/pv root directories
+   * for the test. Creates the operator and domain if its not running.
+   *
+   * @throws Exception if result/pv/operator/domain creation fails
+   */
+  @BeforeEach
+  public void prepare() throws Exception {
+    // initialize test properties and create the directories
+    if (FULLTEST) {
+      LoggerHelper.getLocal().log(Level.INFO, "Checking if operator1 and domain are running, if not creating");
+      createResultAndPvDirs(testClassName);
+      // create operator1
+      if (operator1 == null) {
+        ArrayList<String> targetDomainsNS = new ArrayList<String>();
+        targetDomainsNS.add(domainNSShutOpCluster);
+        targetDomainsNS.add(domainNSShutOpDomain);
+        targetDomainsNS.add(domainNSShutOpEnv);
+        targetDomainsNS.add(domainNSShutOpOverrideViaEnv);
+        Map<String, Object> operatorMap = createOperatorMap(getNewSuffixCount(), true, testClassName);
+        operatorMap.put("domainNamespaces",targetDomainsNS);
+        operator1 = TestUtils.createOperator(operatorMap, Operator.RestCertType.SELF_SIGNED);
+        Assertions.assertNotNull(operator1);
+        namespaceList.append((String)operatorMap.get("namespace"));
+        namespaceList.append(" ")
+            .append(domainNSShutOpEnv)
+            .append(" ")
+            .append(domainNSShutOpDomain)
+            .append(" ")
+            .append(domainNSShutOpOverrideViaEnv)
+            .append(" ")
+            .append(domainNSShutOpCluster);
+      }
+    }
+  }
+
+  /**
+   * Releases k8s cluster lease, archives result, pv directories.
+   *
+   * @throws Exception if failed to delete the created objects or archive results.
+   */
+  @AfterAll
+  public static void staticUnPrepare() throws Exception {
+    if (FULLTEST) {
+      tearDown(new Object() {
+      }.getClass().getEnclosingClass().getSimpleName(), namespaceList.toString());
+
+      LoggerHelper.getLocal().log(Level.INFO, "SUCCESS");
+    }
+  }
+
+  /**
+   * Start domain with added shutdown options at the domain level
+   * and verify values are propagated to server level.
+   *
+   * @throws Exception if domain cannot be started or failed to verify shutdown options.
+   */
+  @Test
+  public void testAddShutdownOptionsToDomain() throws Exception {
+
+    Assumptions.assumeTrue(FULLTEST);
+    String testMethodName = new Object() {
+    }.getClass().getEnclosingMethod().getName();
+    logTestBegin(testMethodName);
+    Map<String, Object> shutdownProps = new HashMap<>();
```

Comment:
remove the empty diamond

Reply:
Isn't the empty diamond correct?

Reply:
You are right, uses an unsafe cast to a HashMap without a diamond. Please ignore my comment.